processing/tagger.py

```diff
@@ -0,0 +1,92 @@
+from operator import itemgetter
+from itertools import groupby
+
+DEFAULT_CONFIDENCE = 0.85
+FALSE_POSITIVE = "false-positive"
+UNIDENTIFIED = "unidentified"
+MULTIPLE = "multiple animals"
+STATUS = "status"
+CLARITY = "clarity"
+
+MESSAGE = 'message'
+CONFIDENCE = 'confidence'
+FALSE_POSITIVE_TAG = {"event": "false positive", CONFIDENCE: DEFAULT_CONFIDENCE}
+
+
+def calculate_tags(tracks, conf):
+  # No tracks found so tag as FALSE_POSITIVE
```

Comment:
For some reason all the indents in this file are 2 spaces instead of the customary 4